### ISSUE TITLE
paper

### ISSUE THREAD

**Issue by Zhezhe-Zhang** [dejanpan/dji_phantom_plant_counting#3]
Is there a paper corresponding to this code?

**Comment by dejanpan** [comment]
No, unfortunately not. Implementation was all based on known algorithms. 

Do you have any specific questions?
